wordpress-mobile/WordPress-iOS
# Update self hosted login prompt

**Fixes** #8636 

**To test:**

From a clean install trigger LogIn and check that the screen matches the design.

![simulator screen shot - iphone 7 - 2018-02-20 at 08 03 52](https://user-images.githubusercontent.com/5558824/36420727-ad8cbf06-1614-11e8-9d2f-f98c2bdb2278.png)

CC: @folletto
